src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx

```diff
@@ -120,19 +120,27 @@ class IntroductionPage extends React.Component {
               .
             </p>
           )}
-          <ol>
-            <li className="process-step list-one">
-              <h3 className="vads-u-font-size--h4">Prepare</h3>
+
+          <va-process-list class="vads-u-padding-y--0">
+            {/* Prepare */}
+            <va-process-list-item>
+              <h3 className="vads-u-padding-top--0">Prepare</h3>
```

Comment:
bummer we don't know where the `m-process-list` styling is coming from. i tried removing the imports from `src/applications/disability-benefits/all-claims/sass/disability-benefits.scss` but it's still being included somewhere.